src/site/content/en/blog/native-lazy-loading/index.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Native lazy loading for the web
+subhead: Browser-level native lazy loading is finally here!
+authors:
+  - houssein
+  - addyosmani
+  - mathiasbynens
+date: 2019-06-20
+hero: hero.png
+alt: Phone outline with loading image and assets
+description: |
+  This post covers the loading attribute and how it can be used
+  to defer the loading of images and iframes
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+Support for natively lazy-loading images and iframes is coming to the web!
+
+<figure class="w-figure w-figure--fullbleed">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="./lazyload.webm" type="video/webm">
+    <source src="./lazyload.mp4" type="video/mp4">
+  </video>
+ <figcaption class="w-figcaption w-figcaption--fullbleed">
+    <a href="https://mathiasbynens.be/demo/img-loading-lazy">Lazy loading demo</a>
+  </figcaption>
+</figure>
+
+Starting with Chrome 76, you’ll be able to use the new `loading` attribute to lazy-load resources,
+without the need for a separate JavaScript library. Let’s dive into the details.
+
+## Motivation
+
+According to [HTTPArchive](https://httparchive.org/reports/page-weight), images are the most
+requested type of asset for a majority of websites and usually take up more bandwidth than any other
+resource. At the 90th percentile, sites send down ~4.7 MB of images on desktop & mobile. That’s a
+lot of cat pictures.
+
+Embedded inline frames use a lot of data too and can also harm page performance. Only loading
+non-critical, below-the-fold images and iframes when the user is likely to see them will improve
+page load times, minimize user bandwidth and reduce memory usage.
+
+Currently, there are two ways to defer the loading of off-screen images and inline frames:
+
+- Using the [Intersection Observer
+  API](https://developers.google.com/web/updates/2016/04/intersectionobserver)
+- Using [`scroll`, `resize`, or `orientationchange` event
+  listeners](https://developers.google.com/web/fundamentals/performance/lazy-loading-guidance/images-and-video/#using_event_handlers_the_most_compatible_way)
+
+Either option can let developers include lazy loading functionality, and many have built third-party
+libraries to provide even simpler-to-use abstractions. With lazy loading supported directly by the
+browser however, an external library would not need to be loaded and relied upon. This will also
+ensure that deferred loading of images and frames will still work even if JavaScript is disabled on
```

Comment:
Slightly simpler:

> This will also ensure that deferred loading of images and frames will still work

--> 

> This also ensures that deferred loading of images and frames still works

(in general, "avoid will" is a pretty good advice)